swupd bundle-info python3-urllib3
Error: Cannot acquire lock file, another swupd process is already running (possibly auto-update)
Error: Failed swupd initialization, exiting now
